Tests/Rules/EnvironmentEntryTests.swift

```diff
@@ -271,4 +271,97 @@ final class EnvironmentEntryTests: XCTestCase {
         """
         testFormatting(for: input, rule: .environmentEntry, options: FormatOptions(swiftVersion: "6.0"))
     }
+
+    func testEntryMacroReplacementWhenDefaultValueIsNotComputed() {
+        let input = """
+        struct ScreenStyleEnvironmentKey: EnvironmentKey {
+            static var defaultValue: ScreenStyle = ScreenStyle()
```

Comment:
It would also be good to have a test case where the property doesn't have an explicit type annotation, I don't see an example like this in your existing tests 

```suggestion
            static let defaultValue = ScreenStyle()
```